Conference-Android/app/src/main/java/com/sagetech/conference_android/app/ui/viewModel/EventDetailViewModel.java

```diff
@@ -14,118 +19,93 @@
-    }
-
-    public void setDescription(String description) {
-        this.description = description;
+        return sessionData.getFullDesc();
     }
 
     public String getRoomName() {
-        return roomName;
-    }
-
-    public void setRoomName(String roomName) {
-        this.roomName = roomName;
+        return roomData.getShortDesc();
     }
 
     public Date getStartDttm() {
-        return startDttm;
-    }
-
-    public void setStartDttm(Date startDttm) {
-        this.startDttm = startDttm;
+        return sessionData.getStartDttm();
     }
 
     public Date getEndDttm() {
-        return endDttm;
+        Calendar cal = Calendar.getInstance();
+        cal.setTime(sessionData.getStartDttm());
+        cal.add(Calendar.MINUTE, sessionData.getDurationMinutes());
+        return cal.getTime();
     }
 
-    public void setEndDttm(Date endDttm) {
-        this.endDttm = endDttm;
-    }
+    public List<EventDetailPresenterView> getPresenters() {
+        List<EventDetailPresenterView> views = new ArrayList<EventDetailPresenterView>();
 
-    public EventType getEventType() {
-        return eventType;
-    }
+        for (PresenterData presenter : presenterDatas) {
+            if (presenter == null) {
+                continue;
+            }
+            views.add(new EventDetailPresenterView(presenter));
+        }
 
-    public void setEventType(EventType eventType) {
-        this.eventType = eventType;
+        return views;
     }
 
     public class EventDetailPresenterView {
-        private String firstName;
-        private String lastName;
-        private String biography;
 
-        public String getFullName() {
-            return String.format("%s %s", firstName, lastName);
-        }
+        private PresenterData presenterData;
 
-        public String getFirstName() {
-            return firstName;
+        public EventDetailPresenterView(PresenterData presenterData) {
+            this.presenterData = presenterData;
         }
 
-        public void setFirstName(String firstName) {
-            this.firstName = firstName;
+        public String getFullName() {
+            return String.format("%s %s", this.getFirstName(), this.getLastName());
         }
 
-        public String getLastName() {
-            return lastName;
+        public String getFirstName() {
+            return presenterData.getId() + "";
         }
 
-        public void setLastName(String lastName) {
-            this.lastName = lastName;
+        public String getLastName() {
+            return presenterData.getUserId();
         }
 
         public String getBiography() {
-            return biography;
+            return presenterData.getShortBio();
         }
 
-        public void setBiography(String biography) {
-            this.biography = biography;
+        public String getPicUrl() {
+            return "https://dl.dropboxusercontent.com/s/94j05poa7rg2rf1/UnknownProfile.png";  //TODO: pic url when available
```

Comment:
Attempt to get this value from a value coming back from the api, prior to using the default value. That way, when the data does come back, we will not have to change this code again.